src/app.config.js

```diff
@@ -53,6 +53,16 @@ export const app_config = {
    */
   retrocompatibilityAOS6: true,
 
+  /*
+   * This setting enables minimal required mobile app version
+   */
```

Comment:
Add more details please, when this is used, what it does and what happened when disabled or no links.